plans/eventual_consistency.pp

```diff
@@ -16,31 +16,33 @@
   # Wait for approval if the environment is protected
   cd4pe_deployments::wait_for_approval($target_environment) |String $url| { }
 
+  # If this is a module deployment, the update ref may have fail because we get $repo_target_branch from the node
+  # group name and it may not exist on the module repository. It's best to ensure the branch exists by calling
+  # create_git_branch(). This is a safe operation beceause the function is idempotent
+  if($repo_type == 'MODULE'){
+    $git_branch_cleanup = false
```

Comment:
Just for clarity, do we want feature branches to stick around after the deployment?

Reply:
That's what we shipped in 3.0. I think that's a change in behavior from 2.18 though?